client/components/addHeroForm/AddHeroForm.js

```diff
@@ -84,8 +133,13 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
+AddHeroForm.defaultProps = {
+  acts: PropTypes.arrayOf(PropTypes.object),
```

Comment:
in defaultProps u can do just  `acts: []` but i'm not sure that this is correct

Reply:
oops, thank you! :D